test/e2e-test-application/src/app/project/project.component.html

```diff
@@ -274,264 +193,167 @@ <h3 class="fd-layout-panel__title">Navigate</h3>
+            <app-code-snippet data="linkManager().navigate('/settings', null, true)"></app-code-snippet>
           </li>
           <li class="fd-list__item">
-            <a
-              href="javascript:void(0)"
-              class="fd-link"
-              (click)="
+            <a href="javascript:void(0)" class="fd-link" (click)="
                 linkManager()
                   .withParams({ test: true })
                   .navigate('/settings', null, true)
-              "
-            >
+              ">
               extended preserved view example with params: project to global
-              settings and back</a
-            >
-            <app-code-snippet
-              data="linkManager().withParams({test: true}).navigate('/settings', null, true)"
-            >
+              settings and back</a>
+            <app-code-snippet data="linkManager().withParams({test: true}).navigate('/settings', null, true)">
             </app-code-snippet>
           </li>
           <li class="fd-list__item">
-            <a
-              href="javascript:void(0)"
-              class="fd-link"
-              (click)="
+            <a href="javascript:void(0)" class="fd-link" (click)="
                 linkManager()
                   .withParams({ test: true })
-                  .openAsModal('/settings', {
-                    title: 'Preserved View',
-                    size: 'm'
+                  .openAsModal('/settings', {title: 'Preserved View',size: 'm'
                   })
-              "
-            >
+              ">
               extended preserved view example with params rendered in a modal:
-              project to global settings and back</a
-            >
+              project to global settings and back</a>
             <app-code-snippet
-              data="linkManager().withParams({test: true}).openAsModal('/settings', {title:'PreservedView', size:'m'})"
-            >
+              data="linkManager().withParams({test: true}).openAsModal('/settings', {title:'PreservedView', size:'m'})">
+            </app-code-snippet>
+          </li>
+          <li class="fd-list__item">
+            <a href="javascript:void(0)" class="fd-link" (click)="
+              linkManager().openAsDrawer('/projects/pr2/drawer')">
+              Open drawer with default settings</a>
+            <app-code-snippet data="linkManager().openAsDrawer('/projects/pr1/drawer');">
+            </app-code-snippet>
+          </li>
+          <li class="fd-list__item">
+            <a href="javascript:void(0)" class="fd-link" (click)="
+              linkManager().openAsDrawer('/projects/pr1/drawer', {header: false, size: 'xs' ,backdrop: true})">
+              Open view in drawer component with and w/o backdrap and header</a>
+            <app-code-snippet data="linkManager().openAsDrawer('/projects/pr1/drawer', {header: false, size:'xs'
+                backdrop: true});">
```

Comment:
just a comma here after xs 
```suggestion
            <app-code-snippet data="linkManager().openAsDrawer('/projects/pr1/drawer', {header: false, size:'xs', backdrop: true});">
```